### DISCUSSION TITLE
Welcome to Manga Watcher X android ,Discussions!

### DISCUSSION THREAD

**Discussion by veopot** [veopot/mwx-json#2] [General]
Пожалуйста без матов ,
Если у вас возникла проблема с использованием приложения MangaWatcher, то, пожалуйста, всегда тщательно описывайте проблему.

**Comment by veopot** [comment]
Эх ни кого

**Reply by veopot** [reply]
Идет 2023г тут до сих пор пусто!

**Reply by veopot** [reply]
2024 тут пусто

**Reply by AlexB1324** [reply]
День добрый, с недавнего обновления перестал работать парсер мангалиба(не грузит мангу уже добавленную), до обновления все работало
Подскажите что делать?

**Reply by veopot** [reply]
> День добрый, с недавнего обновления перестал работать парсер мангалиба(не грузит мангу уже добавленную), до обновления все работало Подскажите что делать?


Добрый день 14 чесла сайты сети либ полностью обновили движок сайта
Об этом была новость в нашей группе и телеграмме

Обновление у всех приложений подобного рода, не только у нашей сломало чтение с данных сайтов

Так как под капотом у сайта структура сильно изменилась

Было решение полность переписать работу парсера под эти сайты и за чего
Пришлось выпустить новую версию и новые парсеры под данные сайты

Но при этом Внимание: манга что у вас была с сайтов сети либ, в новой версии приложения 3.1.6.9 придется передобавить по новой (Простите)

**Reply by AlexB1324** [reply]
Блин, ну ладно
Спасибо)

**Reply by AlexB1324** [reply]
Кстати, ещё есть вопрос.
То, что парсер nhentai при загрузке и попытке чтения манги выдаёт ошибку 403 это же не нормально?
Я своими силами что-то смогу сделать, не меняя код парсера?

**Reply by veopot** [reply]
> Кстати, ещё есть вопрос. То, что парсер nhentai при загрузке и попытке чтения манги выдаёт ошибку 403 это же не нормально? Я своими силами что-то смогу сделать, не меняя код парсера?

Давно не трогал данный парсер надо проверять

**Reply by veopot** [reply]
> Кстати, ещё есть вопрос. То, что парсер nhentai при загрузке и попытке чтения манги выдаёт ошибку 403 это же не нормально? Я своими силами что-то смогу сделать, не меняя код парсера?

Попробовал поправить, попробуйте обновить парсер выложил обновление

**Reply by AlexB1324** [reply]
Здравствуйте, есть вопрос
Можно ли сделать парсер этого сайта (сам сделать не могу, т.к. руки не из того места, поэтому прошу вас)
https://manga.ovh/

**Reply by veopot** [reply]
Пробовал уже но проблема там с тем что у некоторых манг может быть несколько веток перевода 
А в приложении нет выбора веток перевода

**Reply by AlexB1324** [reply]
В мангалибе же такая же тема
Там кто-то другой делал парсер?
Upd. Я дурак, там по-другому

**Reply by veopot** [reply]
В мангалиб по умолчанию если больше глав в переводе он главный в апи вроде

**Reply by AlexB1324** [reply]
А в самом парсере так сделать нельзя?

**Reply by veopot** [reply]
На выходных гляну может у них что-нибудь изменилось за это время
